db/migrate/20231129185914_add_text_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddTextToPosts < ActiveRecord::Migration[7.1]
+  def change
+    add_column :posts, :text, :string
+  end
+end
```

Comment:
- [ ] [optional] Kindly consider removing this migration file, you have already created the text column in the `20231129101210_create_posts` file